packages/data-model/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @iroha2/data-model
 
+## 2.0.0
+
+### Minor Changes
+
+- a99d219: **fix!**: define `exports` field; use `*.cjs` extension for `require()` imports and `*.mjs` for `import`
+
+### Patch Changes
+
+- Updated dependencies [a99d219]
+  - @iroha2/i64-fixnum@0.4.0
```

Comment:
This format is different from the one above (e.g. the one above starts with the commit hash, this one has the hash at the end of line in brackets). I'd recommend following the same structure for all changelog entries

Reply:
It is generated automatically and I'd like to avoid touching this.